geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/analytics/HistogramJob.scala

```diff
@@ -96,16 +101,26 @@ class HistogramJob(args: Args) extends GeoMesaBaseJob(args) {
   val aggregates: UnsortedGrouped[Product, Long] = groups.aggregate(Aggregator.size)
 
   // write out the histogram to a tsv file - we create the tabs ourselves since length isn't known
-  aggregates.toTypedPipe.map { case (group, count) => s"${group.productIterator.mkString("\t")}\t$count" }
+  if (writeToAccumulo) {
+    aggregates.toTypedPipe
+      .flatMap { case (group, count) =>
```

Comment:
Is the flatMap necessary since a Seq is returned?

Is it possible to use a 'map' and return (null: Text, mut)?